contents/monte_carlo_integration/code/rust/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "montecarlo"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rand = "0.8.4"
+
+[[bin]]
+path = "./monte_carlo.rs"
+name = "main"
```

I would use the file name to make things clearer.

```suggestion
name = "monte_carlo"
```

---

The name determines what the name of the executable is. If it's not constant then it will need to be passed as another arguement to the build, and as it gets renamed automatically when it's moved I thought this would be simpler.